src/components/Navigation.jsx

```diff
@@ -0,0 +1,20 @@
+import '../App.css';
+
+const Nav = () => (
+  <nav className="nav">
+    <h1>Math Masters</h1>
+    <ul className="nav-ul">
+      <li>
+        <a href="/">Home</a>
+      </li>
+      <li>
+        <a href="/calculator">Calculator</a>
+      </li>
+      <li>
+        <a href="/quotes">Quotes</a>
+      </li>
+    </ul>
+  </nav>
+);
+
```

Comment:
- [ ] Your page reloads as I navigate through the app. This should not be allowed as we building a single-page application with `react-router-dom`. Instead of using the `<a>` tag, kindly use `<NavLink>` from the router package to fix this. This is an example.

  ```
  import '../App.css';
  import { NavLink } from 'react-router-dom';
  
  const Nav = () => (
    ...
          <NavLink to="/">Home</NavLink>
        ...
          <NavLink to="/calculator">Calculator</NavLink>
        ...
          <NavLink to="/quotes">Quotes</NavLink>
        ...
  );
  ```